### PR TITLE
Disabled user profile image in input

### LINKED ISSUE #72: User profile picture from website-static
Until we have the S3 bucket working, we will use `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/<username>/img.png`

Repo here: https://github.com/Real-Dev-Squad/website-static/

### PR DESCRIPTION
**Pull request by ankushdharkar**
The user profile picture will come from the website-static repo. We will use the username as the key for selecting the profile image

Closes #72

### CODE REVIEW

File: user-form.html [hunk 1]
```diff
@@ -170,7 +170,8 @@ <h2>Account Details</h2>
               type="text"
               id="profile_pic"
               name="profile_pic"
-              placeholder="Link to your profile picture"
+              placeholder="Your profile picture comes from website-static"
```
Comment (swarajpure) [inline user-form.html:173]: What if we could link the `website-static` repo? (For someone who doesn't know what is `website-static`)

Comment (ankushdharkar) [reply user-form.html:173]: Good idea, let's do that as a follow-up feature

This entire www page is quite temporary. We will move it to an Ember app in future

Comment (swarajpure) [reply user-form.html:173]: I'm sure if its possible or no since it's placeholder (plain text I suppose)

Comment (ankushdharkar) [reply user-form.html:173]: Placeholder is just dead text, but we can always provide a link under the input box. I like your idea.

Comment (swarajpure) [reply user-form.html:173]: Cool ⚡

**Review by swarajpure — APPROVED** [review]

File: user-form.html (hunk 1 above)
Comment (ankushdharkar) [reply user-form.html:173]: @swarajpure https://github.com/Real-Dev-Squad/website-www/issues/76

**Pull request merged**